halium-install

```diff
@@ -38,6 +38,35 @@ do_shell()
 	adb shell "$@"
 }
 
+do_chroot()
+{
+	do_shell "chroot /cache/system $@"
+}
+
+check_busybox()
+{
+    export device_architecture=$(do_shell "uname -m")
+    export busybox_exists=$(do_shell "if [ -d /sbin/busybox ]; then echo true; else echo false; fi")
+    mkdir busybox
```

Comment:
move the mkdir in the else part of the next if as we don't need that directory if busybox already exists.